crates/bevy_sprite/src/bundle.rs

```diff
@@ -11,9 +13,9 @@ use bevy_transform::components::{GlobalTransform, Transform};
 ///
 /// # Extra behaviours
 ///
-/// You may add the following components to enable additional behaviours
+/// You may add the following components to enable additional behaviours:
```

Comment:
I think it should be clear whether `ImageScaleMode` and `TextureAtlas` can be used together or not. If not, here it should say "either component" or similar.

Reply:
Currently they can't be used together, but there's a PR to add support: https://github.com/bevyengine/bevy/pull/12059.

Reply:
Added a note.